Makefile

```diff
@@ -1,43 +1,42 @@
 #
-# Makefile for phoenix-rtos-devices
+# Makefile for phoenix-rtos-tests
 #
 # Copyright 2018, 2019 Phoenix Systems
 #
 # %LICENSE%
 #
 
-SIL ?= @
-MAKEFLAGS += --no-print-directory
-
-TARGET ?= ia32-generic
-
 include ../phoenix-rtos-build/Makefile.common
+# FIXME: this include should be done by Makefile.common
 include ../phoenix-rtos-build/Makefile.$(TARGET_SUFF)
 
-INSTALL_PROGS :=
-
-.PHONY: clean
-clean:
-	@echo "rm -rf $(BUILD_DIR)"
-
-ifneq ($(filter clean,$(MAKECMDGOALS)),)
-	$(shell rm -rf $(BUILD_DIR))
-endif
+.DEFAULT_GOAL := all
 
-T1 := $(filter-out clean all install,$(MAKECMDGOALS))
-ifneq ($(T1),)
-	include $(T1)/Makefile
-.PHONY: $(T1)
-$(T1): all
-else
-	include Makefile.$(TARGET_SUFF)
+ifneq ($(filter %clean,$(MAKECMDGOALS)),)
+$(info cleaning targets, make parallelism disabled)
+.NOTPARALLEL:
 endif
```

Comment:
What do you think about moving this to Makefile.common?

Reply:
yeah, good idea, I'll add it in phoenix-rtos-build and will be removing from other Makefiles with other updates.